base/src/test/java/org/kontinuity/catapult/base/identity/IdentityVisitorTest.java

```diff
@@ -0,0 +1,62 @@
+package org.kontinuity.catapult.base.identity;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.fail;
+
+/**
+ * @author <a href="mailto:ggastald@redhat.com">George Gastaldi</a>
+ */
+public class IdentityVisitorTest {
+
+    @Test
+    public void testTokenIdentity() {
+        final AtomicBoolean test = new AtomicBoolean();
```

Comment:
No, just something to be accessed from the inner class because I can´t use the primitive here